intro-HPC/title.tex

```diff
@@ -81,7 +81,9 @@
 How to start jobs with user interaction?                                & \strong{\ref{ch:running-interactive-jobs}} & \nameref{ch:running-interactive-jobs} \\ \hline
 Where do the input and output go? Where to collect my results?          & \strong{\ref{ch:running-jobs-with-input-output-data}} & \nameref{ch:running-jobs-with-input-output-data} \\ \hline
 Can I speed up my program by exploring parallel programming techniques? & \strong{\ref{ch:multi-core-jobs-parallel-computing}} & \nameref{ch:multi-core-jobs-parallel-computing} \\ \hline
+\ifthenelse{\not \boolean{antwerpen}}{
```

Comment:
Same here, this should be under `\ifgent`, we overlooked that...

Reply:
Since neither colleagues from Leuven or Brussels removed it from their version, I thought it should stay there.
So no objection removing it. I will update both files.